pkg/util/echoutil/middleware_zap_logger.go

```diff
@@ -0,0 +1,55 @@
+package echoutil
+
+import (
+	"time"
+
+	"github.com/labstack/echo"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func ZapLoggerMiddleware(log *zap.Logger) echo.MiddlewareFunc {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ZapLoggerMiddleware should have comment or be unexported